src/components/molecules/customers-groups-summary/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
```

Comment:
nit: this is not yet used - is the usage coming in another PR?

Reply:
It's used in: `src/components/templates/customer-group-table/config.tsx`

Reply:
Sorry my bad!